src/model_theory/satisfiability.lean

```diff
@@ -186,6 +188,64 @@ begin
     rw [← lift_le.{_ (max (max u w') v w)}, lift_lift, lift_lift, lift_lift, lift_lift] }
 end
 
+theorem exists_elementarily_equivalent_card_eq_of_le (M : Type w') [L.Structure M] [nonempty M]
+  (κ : cardinal.{w})
+  (h1 : ℵ₀ ≤ κ)
+  (h2 : cardinal.lift.{w} L.card ≤ cardinal.lift.{max u v} κ)
+  (h3 : cardinal.lift.{w'} κ ≤ cardinal.lift.{w} (# M)) :
+  ∃ (N : category_theory.bundled L.Structure), M ≅[L] N ∧ # N = κ :=
+begin
+  obtain ⟨S, _, hS⟩ := exists_elementary_substructure_card_eq L ∅ κ h1 (by simp) h2 h3,
+  haveI : small.{w} S,
+  { rw [← lift_inj.{_ (w + 1)}, lift_lift, lift_lift] at hS,
+    exact small_iff_lift_mk_lt_univ.2 (lt_of_eq_of_lt hS κ.lift_lt_univ') },
+  refine ⟨(equiv_shrink _).bundled_induced L, S.elementarily_equivalent.symm.trans
+      (equiv.bundled_induced_equiv L _).elementarily_equivalent, lift_inj.1 (trans _ hS)⟩,
+  simp only [equiv.bundled_induced_α, lift_mk_shrink'],
+end
+
+theorem exists_elementarily_equivalent_card_eq_of_ge (M : Type w') [L.Structure M] [infinite M]
+  (κ : cardinal.{w})
+  (h1 : lift.{w} L.card ≤ cardinal.lift.{max u v} κ)
+  (h2 : lift.{w} (# M) ≤ cardinal.lift.{w'} κ) :
+  ∃ (N : category_theory.bundled L.Structure), M ≅[L] N ∧ # N = κ :=
```

Comment:
Does this theorem show that the universe restriction in `exists_elementary_embedding_card_eq` is unnecessarily restrictive, because you can get `N` in any universe of the form `(N : bundled.{max w w'} L.Structure)`? Maybe you can prove that common generalization? 
Or if you never expect to use `exists_elementary_embedding_card_eq` after this, should this be tagged with the name ULS theorem?

Reply:
I've updated the ULS theorem to deal with universes better, and that allowed me to use one fewer lemma.